buildSrc/build.gradle.kts

```diff
@@ -8,10 +8,12 @@ val detektVersion = "1.23.5"
 repositories {
     mavenCentral()
     gradlePluginPortal()
+    maven("https://jitpack.io")
 }
 
 
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
     implementation("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:$detektVersion")
+    implementation("org.glavo:gjavah:0.3.1")
```

Comment:
Move to version specifier to `Versions` file

Reply:
This cannot be moved there (because these are dependencies for the Gradle script, not the project). Moved the version near `kotlinVersion` and `detektVersion`.